src/ui/public/courier/fetch/request/request_fetch_params_to_body_provider.js

```diff
@@ -0,0 +1,15 @@
+import { requestFetchParamsToBody } from './request_fetch_params_to_body';
+
+export function RequestFetchParamsToBodyProvider(Promise, timefilter, kbnIndex, sessionId) {
+  const timeBounds = timefilter.getActiveBounds();
```

Comment:
Moving the `timefilter.getActiveBounds()` call to the "Provider" is essentially caching and using these values for all requests because the Provider code is only called when it's first imported via the `Private` injector service.

Reply:
Good catch!  I tried to see how this could represent itself in the form of a bug so I could add a test case for it, and as it turns out, these timeBounds aren't even needed anymore.  They are ony used in this call `indexList.toIndexList(timeBounds.min, timeBounds.max)` which uses them only if `isTimeBasedInterval` returns true.  Since time based interval indexes were removed about 5 months ago, that never returns true.

I'm going to remove the logic that needs this.

Reply:
As it turns out this is used for very old indexes that used time based intervals.  Support has been removed for it, but the code exists because we never offered an upgrade mechanism.  https://github.com/elastic/kibana/issues/12242